docs/reference/mapping.asciidoc

```diff
@@ -5,30 +5,61 @@
+`_source` field, which customize how a document's associated metadata is
+handled.
+
+Use _dynamic mapping_ and _explicit mapping_ to define your data. Each method
+provides different benefits based on where you are in your data journey. For
+example, explicitly map fields where you don't want to use the defaults, or to
+gain greater control over which fields are created. You can then allow {es} to
+add other fields dynamically.
+
+NOTE: Before 7.0.0, the mapping definition included a type name.
+{es} 7.0.0 and later no longer accept a _default_ mapping. See <<removal-of-types>>.
+
+.Experiment with mapping options
+****
+<<runtime-search-request,Define runtime fields in a search request>> to
+experiment with different mapping options, and also fix mistakes in your index
+mapping values by overriding values in the mapping during the search request.
+****
 
-* which string fields should be treated as full text fields.
-* which fields contain numbers, dates, or geolocations.
-* the <<mapping-date-format,format>> of date values.
-* custom rules to control the mapping for
-  <<dynamic-mapping,dynamically added fields>>.
-
-A mapping definition includes metadata fields and fields:
-
-<<mapping-fields,Metadata fields>>::
-
-Metadata fields are used to customize how a document's associated metadata is
-treated. Examples of metadata fields include the document's
-<<mapping-index-field,`_index`>>, <<mapping-id-field,`_id`>>, and
-<<mapping-source-field,`_source`>> fields.
+[discrete]
+[[mapping-dynamic]]
+== Dynamic mapping
+<<dynamic-field-mapping,Dynamic mapping>> allows you to experiment with
+and explore data when you’re just getting started. {es} adds new fields
+automatically, just by indexing a document. You can add fields to the top-level
+mapping, and to inner <<object,`object`>>  and <<nested,`nested`>> fields.
 
-<<mapping-types,Fields>>::
+Use <<dynamic-templates,dynamic templates>> to define custom mappings that are
+applied to dynamically added fields based on the matching condition. You can
+<<match-mapping-runtime-fields,map runtime fields in a dynamic template>>
+so that new fields are automatically added to the index mapping as runtime
+fields.
 
-A mapping contains a list of fields or `properties` pertinent to the
-document. Each field has its own <<mapping-types, data type>>.
+[discrete]
+[[mapping-explicit]]
+== Explicit mapping
+<<explicit-mapping,Explicit mapping>> allows you to precisely choose how to
+define the mapping definition, such as:
+
+* Which string fields should be treated as full text fields.
+* Which fields contain numbers, dates, or geolocations.
+* The <<mapping-date-format,format>> of date values.
+* Custom rules to control the mapping for
+  <<dynamic-mapping,dynamically added fields>>.
 
-NOTE: Before 7.0.0, the 'mappings' definition used to include a type name.
-For more details, please see <<removal-of-types>>.
+<<runtime-mapping-fields,Explicitly mapping runtime fields>> allows you to make
```

Comment:
If we want to call out Runtime fields explicitly here, I would replace "explicitly mapping runtime fields" with "Runtime fields".

Reply:
Changing to:
>Use runtime fields to make schema changes without reindexing.